### PR TITLE
Document Voice Call for AuthN API Resolves: OKTA-109300

### PR DESCRIPTION
**Pull request by gauthamgowrishankar-okta**
This change will add AuthN API documentation for Voice Call Factor .  The documentation has been added for the following flows

Enrollment 
Activation 
Verification
Password Reset Flows

### CODE REVIEW

**Comment by gauthamgowrishankar-okta** [comment]
@mystiberry-okta  @srinivasanagandla-okta  Please review this change

File: _source/_docs/api/resources/authn.md [hunk 1]
```diff
@@ -2434,13 +2643,11 @@ curl -v -X POST \
 }
 ~~~
 
-#### Verify TOTP Factor
+#### Verify Call Factor
```
Comment (srinivasanagandla-okta) [inline _source/_docs/api/resources/authn.md]: for some reason it looks like we are changing the existing section. can you add the call factor related documentation as the last one so that it treats as new content?

Comment (gauthamgowrishankar-okta) [inline _source/_docs/api/resources/authn.md]: sure @srinivasanagandla-okta  will do that

**Comment by mystiberry-okta** [comment]
Are we still waiting for changes or can I merge, @gauthamgowrishankar-okta ? Thanks!

**Comment by gauthamgowrishankar-okta** [comment]
@mystiberry-okta  need to make a small change, will let u know once thats done

**Comment by mystiberry-okta** [comment]
@gauthamgowrishankar-okta  Can you resolve the conflict so I can include in today's release? Thanks!

**Comment by gauthamgowrishankar-okta** [comment]
@mystiberry-okta  have made  the changes, waiting for @srinivasanagandla-okta  to review, once he gives a green we should be good to go

**Comment by mystiberry-okta** [comment]
@gauthamgowrishankar-okta You have to resolve the conflict before I can merge.

**Comment by gauthamgowrishankar-okta** [comment]
@mystiberry-okta  done removing the conflicts, sorry i misunderstood

File: _source/_docs/api/resources/authn.md [hunk 2]
```diff
@@ -1905,6 +2035,84 @@ curl -v -X POST \
...
+Activates a `call` factor by verifying the OTP.  The request/response is identical to [activating a TOTP factor](#activate-totp-factor)
+
+##### Request Parameters
+{:.api .api-request .api-request-params}
+
+Parameter    | Description                                         | Param Type | DataType | Required | Default
+------------ | --------------------------------------------------- | ---------- | -------- | -------- | -------
+fid          | `id` of factor returned from enrollment             | URL        | String   | TRUE     |
+stateToken   | [state token](#state-token) for current transaction | Body       | String   | TRUE     |
+passCode     | OTP sent to mobile device                           | Body       | String   | TRUE     |
```
Comment (srinivasanagandla-okta) [inline _source/_docs/api/resources/authn.md]: It will be relevant to say "Passcode received via the voice call" instead of "OTP sent to mobile device"

File: _source/_docs/api/resources/authn.md [hunk 3]
```diff
@@ -3933,6 +4360,174 @@ curl -v -X POST \
...
+
+Verifies a Voice Call OTP (`passCode`) made to the user's mobile phone for primary authentication for a recovery transaction with `RECOVERY_CHALLENGE` status.
+
+##### Request Parameters
+{:.api .api-request .api-request-params}
+
+Parameter    | Description                                                  | Param Type | DataType | Required | Default
+------------ | ------------------------------------------------------------ | ---------- | -------- | -------- | -------
+stateToken   | [state token](#state-token) for current recovery transaction | Body       | String   | TRUE     |
+passCode     | OTP sent to device                                           | Body       | String   | TRUE     |
```
Comment (srinivasanagandla-okta) [inline _source/_docs/api/resources/authn.md]: same comment as the previous one about the description for passcode.

File: _source/_docs/api/resources/authn.md [hunk 4]
```diff
@@ -3933,6 +4360,174 @@ curl -v -X POST \
...
+      }
+    }
+  }
+  
+###### Resend Call Recovery Challenge
+{:.api .api-operation}
+
+<span class="api-uri-template api-uri-post"><span class="api-label">POST</span> /authn/recovery/factors/call/resend</span>
+
+Resends a Voice Call with OTP (`passCode`) to the user's mobile phone
```
Comment (srinivasanagandla-okta) [inline _source/_docs/api/resources/authn.md]: We should avoid using the term "mobile" as it doesn't need to be mobile phone.

Comment (mystiberry-okta) [inline _source/_docs/api/resources/authn.md]: I can change to "device", @srinivasanagandla-okta or @gauthamgowrishankar-okta

Comment (gauthamgowrishankar-okta) [inline _source/_docs/api/resources/authn.md]: @mystiberry-okta  sure

File: _source/_docs/api/resources/authn.md (hunk 2 above)
Comment (gauthamgowrishankar-okta) [inline _source/_docs/api/resources/authn.md]: @mystiberry-okta  can you rephrase this as well with your change

**Comment by gauthamgowrishankar-okta** [comment]
@srinivasanagandla-okta  made the changes you suggested

**Comment by mystiberry-okta** [comment]
@gauthamgowrishankar-okta as we discussed, send/sent is fine in the contexts you pointed out. I hope these changes are correct!

File: _source/_docs/api/resources/authn.md [hunk 5]
```diff
@@ -4195,7 +4195,7 @@ curl -v -X POST \
 
 {% api_operation post /authn/recovery/factors/sms/verify %}
 
-Verifies a SMS OTP (`passCode`) sent to the user's mobile phone for primary authentication for a recovery transaction with `RECOVERY_CHALLENGE` status.
+Verifies a SMS OTP (`passCode`) sent to the user's device for primary authentication for a recovery transaction with `RECOVERY_CHALLENGE` status.
```
Comment (gauthamgowrishankar-okta) [inline _source/_docs/api/resources/authn.md]: @mystiberry-okta  for "SMS" factor we want it to me mobile phone, since we dont enroll landlines, "device" would only apply for "Voice Call" Factor

**Comment by srinivasanagandla-okta** [comment]
Looks good. 🚢 

I feel verifying the changes is getting bit of challenge because of the way github decided to show the differences.

@mystiberry-okta @tthompson-okta Is there a way to generate (but not release/deploy) the final HTML based on the topic branch so that we can review how it looks (when users see this on developer.okta.com).
